lib/dogma/formatter/emacs.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Dogma.Formatter.Emacs do
```

Comment:
I *think* that the format comes from Flycheck, so I would prefer that. My second favourite is Emacs. What's your preference?